tests/test_format.py

```diff
@@ -62,7 +62,10 @@
     os.path.join(PYINCORE_ROOT_FOLDER, 'pyincore/analyses/housingrecovery/housingrecovery.py'),
     os.path.join(PYINCORE_ROOT_FOLDER, 'pyincore/analyses/housingrecovery/housingrecoveryutil.py'),
     os.path.join(PYINCORE_ROOT_FOLDER, 'tests/pyincore/analyses/housingrecovery/test_housingrecovery.py'),
-
+    os.path.join(PYINCORE_ROOT_FOLDER, 'pyincore/analyses/waterfacilityrestoration/waterfacilityrestoration.py'),
+    os.path.join(PYINCORE_ROOT_FOLDER, 'pyincore/analyses/waterfacilityrestoration/waterfacilityrestorationutil.py'),
+    os.path.join(PYINCORE_ROOT_FOLDER,
+                 'tests/pyincore/analyses/waterfacilityrestoration/test_waterfacilityrestoration.py')
```

Comment:
please add comma at the end of the line

Reply:
Develop did not have a trailing comma so I followed that pattern. @nunezco2 what is your preference here? Trailing commas are subjective.

Reply:
It has to be separated with the next line by the comma

Reply:
My bad, thought you meant adding a trailing comma at the end of the list. Fixed.